src/components/Header.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Link } from 'gatsby';
+import Logo from '../assets/logo.svg';
+
+import {
+  headerText,
+  headerContainer
+} from '../styles/header.module.scss';
+
+const Header = () => (
```

Comment:
maybe this component (and file) could be renamed to HeaderLogo or something along those lines?